content/en/docs/reference/access-authn-authz/certificate-signing-requests.md

```diff
@@ -613,6 +613,249 @@ To test it, change the context to `myuser`:
 kubectl config use-context myuser
 ```
 
+<!-- TODO this should become a task page -->
+## How to delegate signer permissions within a domain {#custom-domain-delegation}
+
+{{< feature-state feature_gate_name="ValidatingAdmissionPolicy" >}}
+
+When working with [custom signers](#custom-signers) that approve, deny and sign 
+[CertificateSigningRequests](/docs/reference/kubernetes-api/authentication-resources/certificate-signing-request-v1/) 
+using the [Kubernetes API](#signer-api), the signer's identity may be given [privileges](#authorization)
+over the entire domain by using trailing wild-cards in the `resourceNames`:
+
+```yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: top-level-csr-approver
+rules:
+- apiGroups:     ["certificates.k8s.io"]
+  resourceNames: ["example.com/*"]
+  resources:     ["signers"]
+  verbs:         ["approve", "sign"]
```

Comment:
Ideally, either:
- mention that RBAC is only one possible authz mechanism
- avoid a focus RBAC and instead talk about what the SubjectAccessReview(s) will look like

Reply:
The main in-tree alternative is webhook authz, but you can combine that with eg https://www.openpolicyagent.org/integrations/kubernetes-authorization/ or https://github.com/awslabs/cedar-access-control-for-k8s

OpenShift has its own [custom authz](https://docs.openshift.com/container-platform/4.16/authentication/using-rbac.html) that extends K8s RBAC.